notediscovery/data/config/config.yaml

```diff
@@ -0,0 +1,23 @@
+app:
+  name: "NoteDiscovery"
+  tagline: "Your Self-Hosted Knowledge Base"
+
+server:
+  host: "0.0.0.0"
+  port: 8000
+  reload: false
+  allowed_origins: ["*"]
+  debug: false
+
+storage:
+  notes_dir: "./data"
+  plugins_dir: "./plugins"
+
+search:
+  enabled: true
+
+authentication:
+  enabled: false
+  secret_key: "e186cc00ab24b7b1c823bae3c675cc24a5868a4772a3da0dd17e924803478141"
+  password_hash: "$2b$12$t/6PGExFzdpU2PUta0iVY.eDQwvu63kH.c/d4bEnnHaQ5CspH1yrG"
+  session_max_age: 604800
```

Comment:
```suggestion
```
We may not need these environment variables in the configuration if we’re not using NoteDiscovery authentication.